lib/cb2/strategies/rolling_window.rb

```diff
@@ -57,13 +57,13 @@ def key(id=nil)
 
   def increment_rolling_window(key)
     t   = Time.now.to_i
-    pipeline = redis.pipelined do
+    pipeline = redis.pipelined do |pipeline|
```

Comment:
I'm not sure if it ever worked before. https://github.com/redis/redis-rb#pipelining